File: src/test/java/test/ListDepartmentMemberIdsTest.java
package test;
import cn.authing.sdk.java.client.ManagementClient;
import cn.authing.sdk.java.dto.*;
import cn.authing.sdk.java.model.ManagementClientOptions;
import cn.authing.sdk.java.util.JsonUtils;

import java.util.Collections;



public class ListDepartmentMemberIdsTest {

    private static final String ACCESS_KEY_ID = "AUTHING_ACCESS_KEY_ID";
    private static final String ACCESS_KEY_SECRET = "AUTHING_ACCESS_KEY_SECRET";

    public static void main(String[] args) throws Throwable {
        ManagementClientOptions clientOptions = new ManagementClientOptions();
        clientOptions.setAccessKeyId(ACCESS_KEY_ID);
        clientOptions.setAccessKeySecret(ACCESS_KEY_SECRET);
        ManagementClient client = new ManagementClient(clientOptions);

        ListDepartmentMemberIdsDto request = new ListDepartmentMemberIdsDto();
        request.setOrganizationCode("organizationCode_2987");
        request.setDepartmentId("departmentId_7668");
        request.setDepartmentIdType("departmentIdType_849");

        UserIdListRespDto response = client.listDepartmentMemberIds(request);
        System.out.println(JsonUtils.serialize(response));
    }

}

File: src/test/go/test/ListDepartmentMemberIdsTest.go
package main

import (
"authing-go-sdk/client"
"authing-go-sdk/dto"
"fmt"
)

func main() {
    options := client.ManagementClientOptions{
        AccessKeyId:     "AUTHING_USERPOOL_ID",
        AccessKeySecret: "AUTHING_USERPOOL_SECRET",
    }
    var err error
    client, err := client.NewClient(&options)
    if err != nil {
        panic(err)
    }
    request := dto.ListDepartmentMemberIdsDto{
        OrganizationCode: "organizationCode_1570",
        DepartmentId: "departmentId_8929",
        DepartmentIdType: "departmentIdType_7044",
    }
    response := client.ListDepartmentMemberIds(&request)
    fmt.Println(response)
}

